docs/adr/0021-move-verification-tests.md

```diff
@@ -0,0 +1,25 @@
+# 1. Avoid listing file counts unless user is authenticated
```

Comment:
The title of the ADR needs to be updated!

Reply:
🤦